e2e/framework/clients/k8s_block.go

```diff
@@ -3,29 +3,30 @@ package clients
 import (
 	"fmt"
 	"github.com/rook/rook/e2e/framework/contracts"
+	"github.com/rook/rook/pkg/model"
 )
 
-type k8sRookBlock struct {
+type k8sBlockOperations struct {
```

Comment:
renaming struct to k8sBlockOperations to blockOperation. Following similar naming pattern for other operator implementations.